wpa_supplicant/wpa_cli_cmds.c

```diff
@@ -260,7 +274,7 @@ static int wpa_cli_cmd_disable_network(struct wpa_ctrl *ctrl, int argc,
 static int wpa_cli_cmd_add_network(struct wpa_ctrl *ctrl, int argc,
 				   char *argv[])
 {
-	return wpa_ctrl_command(ctrl, "ADD_NETWORK");
```

Comment:
Can you do this in a separate commit as a pre-cursor?

Reply:
Ok sure.